ci-operator/config/cri-o/cri-o/cri-o-cri-o-release-1.28.yaml

```diff
@@ -0,0 +1,68 @@
+base_images:
+  base-machine:
+    name: centos
+    namespace: openshift
+    tag: stream9
+  rhel-coreos-9-base:
+    name: "4.15"
+    namespace: ocp
+    tag: rhel-coreos-9
```

Comment:
can you also update the [main.yaml(https://github.com/openshift/release/blob/master/ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml) file to match this?

Reply:
Sure, added